packages/extension/src/ui/features/accountTokens/TokenList.tsx

```diff
@@ -47,22 +48,25 @@ export const TokenList: FC<TokenListProps> = ({
         />
       }
     >
-      {tokens.map((token) => (
-        <TokenListItemContainer
-          key={token.id}
-          account={account}
-          tokenWithBalance={token}
-          variant={variant}
-          showTokenSymbol={showTokenSymbol}
-          onClick={() => {
-            navigate(
-              navigateToSend
-                ? routes.sendToken(token.id)
-                : routes.token(token.id),
-            )
-          }}
-        />
-      ))}
+      <Suspense fallback={<NewTokenButton isLoading />}>
+        {tokens.map((token) => (
+          <TokenListItemContainer
+            key={token.id}
+            account={account}
+            tokenWithBalance={token}
+            variant={variant}
+            showTokenSymbol={showTokenSymbol}
+            onClick={() => {
+              navigate(
+                navigateToSend
+                  ? routes.sendToken(token.id)
+                  : routes.token(token.id),
```

Comment:
When does it trigger `token`? I only see `sendToken` so far in the wallet.

Reply:
When clicking on the token, we either go directly to the "send token" page or go to the "token" page where there is a send button and an activity "coming soon" text. In this PR we updated it so that it goes directly to send token (https://github.com/alephium/extension-wallet/blob/support-token-list/packages/extension/src/ui/features/accountTokens/TokenList.tsx#L27), I thought there is not much value to show the "token" page yet, we can probably reintroduce it after we introduce the token related activities, WDYT?

Reply:
Now I remember it. I agree we could simplify it for now 👍